Formula/gst-python.rb

```diff
@@ -32,15 +33,13 @@ class GstPython < Formula
   end
 
   def install
-    mkdir "build" do
-      system "meson", *std_meson_args, ".."
-      system "ninja", "-v"
-      system "ninja", "install", "-v"
-    end
+    system "meson", "setup", "build", *std_meson_args
```

Comment:
May need to look into auto-detected gobject-introspection overrides directory. Could pass in `-Dpygi-overrides-dir=...` to make sure correct directory is picked.

Reply:
What does `pygi-overrides-dir` do?

Reply:
It's the option to manually set path for `<site-packages>/gi/overrides` directory. Should be similar to what is used in `gexiv2`. https://gitlab.freedesktop.org/gstreamer/gstreamer/-/blob/main/subprojects/gst-python/meson_options.txt

The failure seems to be that the actual override is not installed correctly. Meson currently defaults to subdirectory under `python.get_install_dir`, but this might be part of prefix scheme issue.

Reply:
Or, I guess, which directory should this point to?

Reply:
Maybe something like:
```suggestion
    site_packages = prefix/Language::Python.site_packages("python3.10")
    system "meson", "setup", "build", *std_meson_args, "-Dpygi-overrides-dir=#{site_packages}/gi/overrides"
```

Reply:
Could also set `-Dpython` to make sure correct Python version is used